### PR TITLE
PICARD-1257: Enable cluster button for unclustered files only when applicable

### PR DESCRIPTION
**Pull request by vishichoudhary**
# Summary
Cluster button should be enable only when there is more than one file in unclustered files.

* This is a…
    * [x] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): [PICARD-1257](https://tickets.metabrainz.org/browse/PICARD-1257)

# Solution

# Action

### CODE REVIEW

File: picard/cluster.py [hunk 1]
```diff
@@ -295,15 +295,15 @@ def __init__(self):
...
 
     def add_file(self, file):
         Cluster.add_file(self, file)
-        self.tagger.window.enable_cluster(self.get_num_files() > 0)
+        self.tagger.window.enable_cluster(self.get_num_files() > 1)
 
     def remove_file(self, file):
         Cluster.remove_file(self, file)
-        self.tagger.window.enable_cluster(self.get_num_files() > 0)
+        self.tagger.window.enable_cluster(self.get_num_files() > 1)
```
Comment (zas) [inline picard/cluster.py]: Make a small method doing that please, same code is repeated 3 times

**Comment by vishichoudhary** [comment]
@zas @sambhav . https://github.com/metabrainz/picard/blob/848745d3cf64150e41691de9dcd14016a66f2fc4/picard/tagger.py#L685 
```
if len(objs) <= 1 or self.unclustered_files in objs:
````
Do we need this condition?.

File: picard/cluster.py [hunk 2]
```diff
@@ -295,15 +295,18 @@ def __init__(self):
...
-        self.tagger.window.enable_cluster(self.get_num_files() > 0)
+        self.enable_cluster()
 
     def remove_file(self, file):
         Cluster.remove_file(self, file)
-        self.tagger.window.enable_cluster(self.get_num_files() > 0)
+        self.enable_cluster()
+
+    def enable_cluster(self):
+        self.tagger.window.enable_cluster(self.get_num_files() > 1)
```
[the remark this answers is not in the document]
Comment (vishichoudhary) [reply picard/cluster.py]: I asked it on IRC.
https://chatlogs.metabrainz.org/brainzbot/metabrainz/msg/4155762/
And I make this understanding. I may or may not be correct. Also one more thing currently Picard doesn't cluster single file. Though the button is enabled.

[the remark this answers is not in the document]
Comment (vishichoudhary) [reply picard/cluster.py]: Ok, I will modify it.

**Pull request closed without merging**